ghost/signup-form/package.json

```
@@ -20,7 +20,7 @@
         "dev": "concurrently \"vite --port 6173\" \"vite preview -l silent\" \"vite build --watch\"",
         "preview": "concurrently \"vite preview -l silent\" \"vite build --watch\"",
         "dev:test": "vite build && vite preview --port 6175",
-        "build": "tsc && vite build",
+        "build": "cd ../i18n && yarn build && cd ../signup-form && tsc && vite build",
```

@daniellockyer  not too comfortable with this, but it's not building since the typescript declaration file didn't exist.... Probably a better place to run this? or should we maybe hard track the declaration file in git? I don't see it changing too often unless we have major changes on i18n. 🤔

---

```suggestion
        "prebuild": "yarn workspace @tryghost/i18n build",
        "build": "tsc && vite build",
```

Perhaps @allouis has stronger opinions but we probably need to add `"prepare": "tsc",` to `i18n`'s package.json so it works in CI and then switch this bit to my suggestion above so i18n is always rebuilt prior to building signup-form (for example when publishing the package)